### ISSUE TITLE
Header or Title

### ISSUE THREAD

**Issue by jimjay203** [uxigene/MMM-QRCode#2]
Hi, is it possible to add a header or a title to this module? I'm using it to provide my guests with wifi and i would like to add a title like "scan me for wifi"

**Comment by uxigene** [comment]
Hello @jimjay203, this could be possible. I'll do that when I'll find some time!

**Comment by bert269** [comment]
Hi
Can you please tell me what the image file should be called? I guess is should reside in the modules/MMM-QRCode directory?
Thanks

**Comment by quiroso** [comment]
Could you provide more specifics about how to set up the module? For instance where and how to add the text.
Thank you,
Olger

**Comment by uxigene** [comment]
@quiroso Yeah, I've updated the Readme file. You have to pass the text property in the configuration.

```
config: {
   text: 'Hello world!'
}
```

**Comment by quiroso** [comment]
Thank you!

Olger Quirós

Sent from my iPhone
Please excuse any typo

**Comment by CFenner** [comment]
Header can be added using the module settings `header`.

```
                {
                        module: 'MMM-QRCode',
                        position: 'top_right',
                        header: "my title",
                        config: {
                                text: "my text",
                        }
                },
```

**Comment by uxigene** [comment]
@CFenner "showRaw" will show the encoded text as plain text. This is not a custom text like "scan me for wifi" which is requested.

**Comment by CFenner** [comment]
@evghenix It’s not about showRaw, but the header one level above. I removed it from the code above.
